testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -473,5 +476,12 @@
 PDB_rama = resource_filename(__name__, 'data/19hc.pdb.gz')
 PDB_janin = resource_filename(__name__, 'data/1a28.pdb.gz')
 
+ITP_ala = resource_filename(__name__, 'data/gromacs_ala10.itp')
```

Comment:
this is a sneaky gotcha, but new data files need to get added to the testsuite setup.py so they get bundled up into the tarballs of the release. There’ll be a bunch of glob patterns but I don’t think the new itps will get caught